runtime/src/main/java/org/corfudb/runtime/view/StreamsView.java

```diff
@@ -133,31 +128,28 @@ public long append(@Nonnull Object object, @Nullable TxResolutionInfo conflictIn
                 : runtime.getSequencerView().next(conflictInfo, streamIDs); // Token w/ conflict info
```

Comment:
I think its cleaner to push Line 126 inside the for-loop and get rid of the other sequencer calls.

Reply:
Done.